roles/kubernetes-apps/ansible/templates/nodelocaldns-daemonset.yml.j2

```diff
@@ -42,9 +43,7 @@ spec:
         - -localip
         - {{ nodelocaldns_ip }}
         - -conf
-        - /etc/coredns/Corefile
-        - -upstreamsvc
-        - coredns
+        - /etc/Corefile
```

Comment:
how is this file configured ?

Reply:
Node-cache generates the Corefile from Corefile.base, substituting any templated values in the process.